Godeps/Godeps.json

```diff
@@ -2551,6 +2551,10 @@
 			"Rev": "55146ba61254fdb1c26d65ff3c04bc1611ad73fb"
 		},
 		{
+			"ImportPath": "google.golang.org/api/logging/v2beta1",
```

Comment:
That's gonna be a pain, as event v2beta1 from master doesn't work

But ACK, will do